docs/index.rst

```diff
@@ -67,6 +67,14 @@ locks:
     finally:
         lock.release()
 
+
+    @lock
+    def decorated():
+        open(file_path, "a").write("You're a decorated Jedi!")
```

Comment:
just print the message and avoid the file open

Reply:
Fixed